src/components/Content/Card.js

```diff
@@ -10,7 +10,16 @@ import { Footer } from '../Footer/Footer';
 export const Card = () => {
   return (
     <CardContainer>
-      <CardDetails />
+      <CardDetails>
+        <Text size={'25px'} margin={'0 0 10px 0'} color={'red'}>
+          Title
```

Comment:
You could create a constants file for any string values similar to `colors.js`

Reply:
Will consider these adjustments for future iterations